api/src/main/java/org/openmrs/module/registrationcore/api/mpi/openempi/OpenEmpiImplementation.java

```diff
@@ -0,0 +1,78 @@
+package org.openmrs.module.registrationcore.api.mpi.openempi;
+
+import org.openmrs.Patient;
+import org.openmrs.module.registrationcore.api.mpi.common.*;
+import org.openmrs.module.registrationcore.api.search.PatientAndMatchQuality;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Qualifier;
+
+import java.util.List;
+import java.util.Map;
+
+public class OpenEmpiImplementation implements MpiProvider {
+
+    @Autowired
+    @Qualifier("registrationcore.mpiPatientImporter")
+    private MpiPatientImporter patientImporter;
```

Comment:
@djazayeri Personally I do not like huge classes which do everything. I like have a bunch of classes, where each one have separate responsibilities.
I think that problem is in bean names, we should rename them to emphasize that that beans are used only for OpenEMPI,